orchestrators/FinanceOperations/tests/Unit/Rules/GLAccountMappingRuleTest.php

```
@@ -4,378 +4,175 @@
-                        new class {
-                            public string $transaction_type = 'INVOICE';
-                            public string $account_code = '4000';
-                        }
-                    ];
+        return new class($accountStatuses) implements GLAccountQueryInterface {
+            /**
+             * @param array<string, bool> $accountStatuses
+             */
+            public function __construct(private array $accountStatuses) {}
+
+            public function find(string $tenantId, string $accountCode): ?GLAccountRuleViewInterface
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)